addons/mrp/models/mrp_unbuild.py

```diff
@@ -195,6 +195,8 @@ def action_unbuild(self):
                     taken_quantity = min(needed_quantity, move_line.quantity - qty_already_used[move_line])
                     if taken_quantity:
                         move_line_vals = self._prepare_move_line_vals(move, move_line, taken_quantity)
+                        if move_line.owner_id and move_line.owner_id != move_line.company_id.partner_id:
```

Comment:
Why this condition on the owner who must be different than the company partner ?
I mean, I don't see the point of setting the company partner as the owner but if so the user does thing like that, I don't see why we should remove it ?